src/reconstruction/refinement/addMultipleReactions.m

```diff
@@ -0,0 +1,169 @@
+function newmodel = addMultipleReactions(model,rxnIDs,metList,Stoichiometries,varargin)
+%Add a batch of reactions to the model.
```

Comment:
May you give here a more explicit explanation? Also, maybe refer to `addReaction`.